### DISCUSSION TITLE
[Help]: Option to disable keybindings like left/right button for switching slides

### DISCUSSION THREAD

**Discussion by seleckis** [davidjerleke/embla-carousel#976] [Help] [answered]
### Which variants of Embla Carousel are relevant to this feature request?

- [X] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

I'm using form elements in carousel slides and left/right buttons are needed to move caret in an input. Unfortunately it is not possible because carousel slides are being switched using these buttons. Would be great to have an option to disable key events or a particular key combination.

### Additional Context

Additional details here...

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by davidjerleke** [comment, marked as answer]
@seleckis thanks for your request.

I don't think it's relevant though. Are you using Embla with Shadcn? Because the core Embla library doesn't have any left and right keybindings, but Shadcn does. So you should create your request in their repository instead.

**Reply by seleckis** [reply]
Oh, I didn't expect this. Yes I'm using Shadcn. Now it's clear. Thank you for the quick response.
